cmake/configs/default.cmake

```diff
@@ -471,7 +471,7 @@ hunter_default_version(tacopie VERSION 3.2.0-h1)
 hunter_default_version(tcl VERSION core8.6.8)
 hunter_default_version(termcolor VERSION 1.0.0)
 hunter_default_version(thread-pool-cpp VERSION 1.1.0)
-hunter_default_version(thrift VERSION 0.11.0-p0)
+hunter_default_version(thrift VERSION 0.12.0-p0)
```

Comment:
Doesn't belong here.

> I have checked that this pull request contains only
.travis.yml/appveyor.yml changes or commits merged from the pkg.template branch. All other changes send
to https://github.com/ruslo/hunter. Yes

Reply:
and how should the change from ruslo/hunter be incorporated in the tests?

Reply:
I will merge it after update https://github.com/ruslo/hunter/pull/1908 will be tested. This branch is only for `.travis.yml`/`appveyor.yml` updates.

Reply:
So in the end we can simply skip this pull request because the initial state of this repository has all ready the correct state